tests/unit/test_charm.py

```diff
@@ -566,13 +566,14 @@ def test_relation_broken(self):
             pass
 
     @patch("charm._get_loadbalancer_status", lambda **unused: None)
+    @patch("charm.TraefikIngressCharm._on_config_changed", lambda self, event: None)
```

Comment:
The patch won't be necessary if you add `self.harness.container_pebble_ready("traefik")` after `begin_with_initial_hooks`.
I think that's all I had to do in #100 to make it work.

Reply:
Ah true -- updated. And after you've updated to ops 2.0.0 (when final is out) you can remove all of those too, as `begin_with_initial_hooks()` now calls that.